scripts/run.sh

```diff
@@ -47,7 +47,7 @@ for file in $FILES_MPI; do
     fi
     echo "NUM_PROC: " $NUM_PROC
 
-    mpirun -np $NUM_PROC $file --gtest_repeat=10 || exit 1
```

Comment:
what reasons to change gtest repeat? I use it for catch stochastic problems in parallelism

Reply:
Fixed, no reason, it was changed just accidentally